shopinvader_api_delivery_pickup_colissimo_widget/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Akretion (http://www.akretion.com)
+# Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Shopinvader Pickup Colissimo",
+    "summary": "Shopinvader Pickup Colissimo",
+    "version": "16.0.1.0.0",
+    "category": "e-commerce",
+    "website": "https://github.com/shopinvader/odoo-shopinvader-carrier",
+    "author": "Akretion, Shopinvader",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "external_dependencies": {"python": [], "bin": []},
+    "depends": [
+        "shopinvader_api_delivery_pickup_public_edition",
+        "delivery_roulier_laposte_fr",
```

Comment:
After thinking twice, depending on "delivery_roulier_laposte_fr" is maybe overkill.
We only need this for having the user/password of la poste.

What we can do is the following.
In this module just store the user / password in the ir.config.parameter
laposte.user
laposte.password


Then we can do a glue module with "delivery_roulier_laposte_fr" (this can be done later in an other PR)
That use the user/password stored in the "delivery_carrier_account" https://github.com/OCA/delivery-carrier/blob/16.0/delivery_carrier_account

So in case that you do not print label you do not need to install "delivery_roulier_laposte_fr"